Makefile

```diff
@@ -281,7 +282,11 @@ kind-up kind-down gardener-up gardener-down register-local-env tear-down-local-e
 
 kind2-up kind2-down gardenlet-kind2-up gardenlet-kind2-down: export KUBECONFIG = $(GARDENER_LOCAL2_KUBECONFIG)
 
-kind-extensions-up kind-extensions-down gardener-extensions-up gardener-extensions-down: export KUBECONFIG = $(GARDENER_EXTENSIONS_KUBECONFIG)
+extensions-up: export SEED_KUBECONFIG = $(GARDENER_EXTENSIONS_SEED_KUBECONFIG)
+
+extensions-up: export SEED_HOST = $(shell kubectl get configmaps -n kube-system shoot-info --kubeconfig $(GARDENER_EXTENSIONS_SEED_KUBECONFIG) -o yaml | yq '.data.domain')
```

Comment:
What if the used seed cluster is not a shoot from another Gardener system? Generally, a seed could be any Kubernetes cluster.

Reply:
Now, any kind of Kubernetes clusters could be used as a seed cluster.
When you use a Gardener shoot as seed, some configuration settings are automated.
I tested it on a Gardener shoot and an GKE cluster.